mesop/examples/query_params.py

```diff
@@ -0,0 +1,92 @@
+import mesop as me
+
+
+def on_load(e: me.LoadEvent):
+  me.query_params["on_load"] = "loaded"
+
+
+@me.page(path="/examples/query_params", on_load=on_load)
```

Comment:
[nit] May want to add some tests that include url encoded params.

Reply:
good idea-done.